uber/templates/preregistration/form.html

```diff
@@ -27,14 +27,14 @@
                 }
             });
         {% endif %}
```

Comment:
It seemed to work correctly when I tested it - I only changed it to save having to check the availability in the template, so if we just want an explicit check instead, that's fine.